src/api_manager/check_auth.cc

```diff
@@ -160,10 +160,11 @@ void AuthChecker::Check() {
     Unauthenticated("Missing or invalid credentials");
     return;
   }
-  context_->request()->SetAuthToken(auth_token_);
 
-  env_->LogDebug(std::string("auth token: ") + auth_token_);
+  context_->request()->SetAuthToken(auth_token_);
 
+  //Set auth token for AuthzCache key construction in check_security_rules.cc.
+  context_->SetAuthToken(auth_token_);
```

Comment:
why do you need to set auth token to context_? Can you use the one in context_->request()?